Dockerfile

```diff
@@ -1,10 +1,20 @@
-FROM golang:1-alpine AS build-env
+FROM golang:1.16 AS build
```

Comment:
why hardcoded vs. alpine or latest? not critizicing, just curious.

Reply:
In general, I typically lock in versions where I can.  That gives me a chance to test things locally before committing to an upgrade.  In this case, go only updates twice a year, so it's not much work to occasionally bump the build version.  But of course, it is also probably unnecessarily conservative given the go1 compatibility promise.  I don't have strong feelings on it, and am happy to go either way.

Reply:
that's fine with me as is, I just wanted to understand the thinking, thanks!